### PR TITLE
KFSPTS-30887 add purchasing assistant owner netid

### PR DESCRIPTION
**Pull request by db483**

### CODE REVIEW

File: src/main/java/edu/cornell/kfs/module/purap/document/IWantDocument.java [hunk 1]
```diff
@@ -100,6 +100,8 @@ public class IWantDocument extends FinancialSystemTransactionalDocumentBase impl
     private boolean completed;
     private String processorNetId;
     private String processorName;
+    private String purchasingAssistantNetId;
+    private String purchasingAssistantName;
```
Comment (jhulslander) [inline src/main/java/edu/cornell/kfs/module/purap/document/IWantDocument.java:104]: should this be transient since it isn't persisted?

**Review by jhulslander — COMMENTED** [review]
I had one question on the code,
When opening the I Want document, and the netID field was previously set, should purchasingAssistantName be populated also?  I think IWantDocumentAction.loadDocument() is where we would want to do that.

File: src/main/java/edu/cornell/kfs/module/purap/document/IWantDocument.java (hunk 1 above)
Comment (db483) [reply src/main/java/edu/cornell/kfs/module/purap/document/IWantDocument.java:104]: updated

**Review by jhulslander — APPROVED** [review]
changes look good, I shall merge

**Pull request merged**